src/Cassandra.IntegrationTests/TestClusterManagement/Simulacron/Cluster.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace Cassandra.IntegrationTests.TestClusterManagement.Simulacron
+{
+    public class Cluster : Base
+    {
+        public dynamic Data { get; set; }
+        public List<DataCenter> DataCenters { get; set; }
+        private const string CreateClusterPathFormat = "/cluster?data_centers={0}&cassandra_version={1}&dse_version={2}&name={3}" +
+                                                       "&activity_log={4}&num_tokens={5}";
+
+        public Tuple<string, int> InitialContactPoint
+        {
+            get
+            {
+                var contact = DataCenters.First().Nodes.First().ContactPoint;
+                return GetTupleFromContactPoint(contact);
+            }
+        }
+
+        private Tuple<string, int> GetTupleFromContactPoint(string contact)
+        {
+            if (contact.Contains(":"))
+            {
+                var parts = contact.Split(':');
+                var addr = parts[0];
+                var port = int.Parse(parts[1]);
+                return new Tuple<string, int>(addr, port);
+            }
+            return new Tuple<string, int>(contact, 9042);
+        }
+
+        private Cluster(string id) : base(id)
+        {
+        }
+
+        public static Cluster Create(string dcNodes, string version, string name, bool activityLog, int numTokens, bool dse = false)
```

Comment:
Can we use some kind of `SimulacronOptions` with defaults,  much like `TestClusterManager.CreateNew()` method, to avoid a large number of optional parameters.

In that class, `version` could default to `TestClusterManager.CassandraVersionText`; `name` to some random name, etc...